src/utils.c

```diff
@@ -313,29 +351,19 @@ ts_timestamptz_bucket(PG_FUNCTION_ARGS)
 {
 	Interval   *interval = PG_GETARG_INTERVAL_P(0);
 	TimestampTz timestamp = PG_GETARG_TIMESTAMPTZ(1);
+
+	/*
+	 * USE NARGS and not IS_NULL to differentiate a NULL argument from a call
+	 * with 2 parameters
+	 */
+	Timestamp	origin = (PG_NARGS() > 2 ? PG_GETARG_TIMESTAMPTZ(2) : DEFAULT_ORIGIN);
```

Comment:
Same thing here as above, but in this case it seems origin should be `TimestampTz`